psqlgraph/mocks.py

```diff
@@ -241,7 +241,9 @@ def create(self, label, override=None, all_props=False):
             prop_list = self.schema[label].get('required', [])
 
         for prop in prop_list:
-            if prop == 'type':
+            # these two props are excluded during the real node creation
+            # see `excluded_props` in gdcdatamodel/models/__init__.py
+            if prop in ['id', 'type']:
```

Comment:
Might not be conventional, but maybe we can consider using the excluded_props directly so there's no divergence between the mock and the code...? Otherwise, lgtm.

Reply:
Well, we do not want to have a `gdcdatamodel` dependency in `psqlgraph`. It seems to me that this `excluded_props` business should be handled inside `gdcdatamodel` or the consumer side of the `psqlgraph` as we currently have in the regular code. 
Unfortunately, I have not figured out a nice and clean implementation for that :slightly_frowning_face:

Reply:
Perhaps, we can be a bit more prudent on the `gdcdatamodel` side and include a comment above `excluded_props` to come edit this mock. At least then we have some accountability...? Anyways, just spitballing ideas here.